src/v2/Components/NavBar/Menus/NavBarUserMenu.tsx

```diff
@@ -73,6 +76,16 @@ export const NavBarUserMenu: React.FC = () => {
         <HeartIcon mr={1} aria-hidden="true" /> Saves &amp; Follows
       </NavBarMenuItemLink>
 
+      {isSavedSearchEnabled && (
+        <NavBarMenuItemLink
+          aria-label="View your saved alerts"
+          to="/user/alerts"
+          onClick={trackClick}
+        >
+          <BellIcon mr={1} aria-hidden="true" /> Alerts
+        </NavBarMenuItemLink>
```

Comment:
**question**: I noticed some flashing between server/client renders in the tab bar. If it's tricky to resolve in this PR another option would be to remove the conditional tab/menu renders and have this PR only expose the route, accessible manually via `/user/alerts`.

![2021-11-05 14 14 56](https://user-images.githubusercontent.com/123595/140559102-6cb4328d-1449-4a20-abca-b67cf1b04df5.gif)

Reply:
> it's tricky to resolve in this PR

You are absolutely right. I tried to solve this problem, but in the end nothing helped. If the option to leave only the path `/user/alerts` is suitable, then I will do it 👍

Reply:
@dimatretyak Yes, I think that would be fine for this iteration.